src/Avalonia.Controls.DataGrid/DataGridColumn.cs

```diff
@@ -38,7 +38,7 @@ public abstract class DataGridColumn : AvaloniaObject
         private ICellEditBinding _editBinding;
         private IBinding _clipboardContentBinding;
         private ControlTheme _cellTheme;
-        private readonly Classes _cellStyleClasses = new Classes();
+        private Classes _cellStyleClasses;
```

Comment:
private Classes? _cellStyleClasses;

Reply:
NRT aren't enabled in the DataGrid project yet :( I'm planning to add them in another PR.